content/zh-cn/docs/reference/glossary/dynamic-volume-provisioning.md

```diff
@@ -11,7 +11,6 @@ tags:
 - core-object
 - storage
 ---
-
 <!--
```

Comment:
We don't have strong preference over whether the newline is needed or not.

For instance, [the zh localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#frontmatter-%E7%9A%84%E5%A4%84%E7%90%86) have a newline between.